core/src/main/java/io/substrait/relation/RelCopyOnWriteVisitor.java

```diff
@@ -380,6 +400,24 @@ protected Optional<Expression.SortField> visitSortField(Expression.SortField sor
         .map(expr -> Expression.SortField.builder().from(sortField).expr(expr).build());
   }
 
+  protected Optional<ConsistentPartitionWindow.WindowRelFunctionInvocation> visitWindowRelFunction(
```

Comment:
minor: I would put this directly under the `visit(ConsistentPartitionWindow consistentPartitionWindow)` definition as it's not really a generic utility in the same way that `visitExprList`, in that it operates only on classes found within the `ConsistentPartitionWindow` unlike the the other methods which operate on classes that can exist in multiple relations.